app/templates/components/events/view/overview/general-info.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="header">{{t 'General info'}}</div>
 </div>
 <div class="content">
-  <div class="header"> {{t 'Your event is not live yet. Make it live now'}}</div>
+  <div class="header"> {{t 'Your event is not live yet.  Make it live now'}}</div>
```

Comment:
@CosmicCoder96 This should be dynamic(not live/ live). Use `event.state` for this.